.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,56 @@
+<!-- Thanks for filing an OSCC issue! You're making our world a better place!-->
+
+<!-- Questions about purchasing and pricing for OSCC boards or DriveKit?
+Shoot an email to drivekit@polysync.io -->
+
+
+
+<!--
+Interested in a new feature?
+-->
+
+### Proposed feature
+<!-- What would you like to accomplish? -->
+
+### Use case
+<!-- How would you like to do it? -->
+
+
+
+<!--
+Missing documentaion?
+-->
+
+### Expected Information
```

Comment:
What do we expect people to provide for this field? I'm not sure how this part of the template will help users fill out good bug reports or feature requests.